pset-iterators.js

```diff
@@ -138,6 +160,10 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+const livingPresidents = presidents.filter(president => (president.death_year === null))
+
+console.log(livingPresidents);
+
```

Comment:
@rlou88 Good job here with this one-liner filter method to return all the president's who are currently living.

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L148)